### PR TITLE
Appointments card

### PR DESCRIPTION
**Pull request by Funmi7**
##  What does this PR do
Styles the appointment card

##  Description of task to be completed
Implement a good style for appointments card

##  How should this manually be tested
npm start, login  and go to the dashboard component

##  What are the relevant Trello board stories
As a user, I want to see my appointments

##  Questions (If any):

### CODE REVIEW

**Comment by Funmi7** [comment]
[As a user I want to see my appointments](https://trello.com/c/FqxWRkcU/83-as-a-user-i-want-to-see-my-appointments)

**Comment by Funmi7** [comment]
> Lets try to stay consistent in formatting styled components. e.g break into own file if very long. Also i noticed sometimes styles were below a component and sometimes above, try and stay uniformed. :)

I thought I deleted the ones at the bottom, those stylings weren't mine.

File: src/components/Dashboard.js [hunk 1]
```diff
@@ -41,6 +41,61 @@ function Copyright() {
...
+//           <Box pt={4}>
+//             <Copyright />
+//           </Box>
+//         </Container>
+//       </main>
+//     </div>
+//   );
+// }
+
+// export default Dashboard;
```
Comment (BenjaminGrabow) [inline src/components/Dashboard.js]: Let's delete all comments from our PR's. We don't want to go over every file in the last week, so let us start early with making clean Pull Requests.

File: src/components/DashboardNav.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            prefix={
+              <SearchIcon
+                type='search'
+                style={{
+                  color: 'rgba(0,0,0,.25)',
+                  marginLeft: '3em',
+                }}
+              />
+            }
+          /> */}
```
Comment (BenjaminGrabow) [inline src/components/DashboardNav.js]: Here is also a comment. Everything else seems fine. I will merge this PR after you changed all requested changes. Good work Funmi 🔥🔥

Comment (Funmi7) [reply src/components/DashboardNav.js]: > Here is also a comment. Everything else seems fine. I will merge this PR after you changed all requested changes. Good work Funmi 🔥🔥

Done

**Pull request merged**